Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/change_password_box.tsx

```diff
@@ -137,7 +122,7 @@ export class ChangePasswordBox extends React.Component<Properties> {
           <div style={ChangePasswordBox.STYLE.passwordPadding}/>
           <input type='password' placeholder='Confirm New Password'
             value={this.props.password2}
-            disabled={input2Readonly}
+            disabled={Boolean(this.props.password1 === '')}
```

Comment:
You don't need to wrap a ```bool``` expression in ```Boolean```. Use the expression or cast ```password1``` to ```Boolean```, but don't do both.